pkgs/development/ruby-modules/gem-config/default.nix

```diff
@@ -697,6 +697,16 @@ in
     buildFlags = [ "--disable-lto" ];
   });
 
+  sass-embedded = _attrs: {
+    dontBuild = false;
+    # Skip sass_embedded step, replace exe path with nixpkgs' dart-sass-embedded
+    postPatch = ''
+      substituteInPlace ext/sass/Rakefile \
+        --replace "'embedded.rb' => %w[sass_embedded] " "'embedded.rb'" \
+        --replace "sass_embedded/dart-sass-embedded" "${dart-sass-embedded}/bin/dart-sass-embedded"
+    '';
```

Comment:
In later versions of the gem, these strings no longer appear. This PR really is stale and should be closed.